### DISCUSSION TITLE
Collapse/hide/minimize board columns

### DISCUSSION THREAD

**Discussion by aspearman** [community/community#8445] [Projects and Issues] [answered]
In the table layout, I can collapse the e.g. 'Done' group so I can focus on the other issues. I'd like to have this functionality in the board layout too. 

At the moment I can remove any 'Done' issues using `-status:Done`,  but that still displays an empty 'Done' column. I'd like to minimize the column using the same chevron button in the board layout as in the table layout:

<img width="127" alt="Screen Shot 2021-12-05 at 11 04 25 PM" src="https://user-images.githubusercontent.com/27811981/144742102-5a2f0857-3e92-452f-9415-47235db96b6e.png">

**Comment by interob** [comment]
Would be great if columns can be collapsed, e.g. with the label text vertically:
![image](https://user-images.githubusercontent.com/19994913/159667544-4e199fd6-3fa4-4eee-8414-665c838a0c7b.png)

**Reply by gold2718** [reply]
My board shows a column for the cards that do not have an assigned value for the column field. I do not want to see that column or its cards, even collapsed!

**Reply by KonstantinKlepikov** [reply]
#15241

**Reply by gold2718** [reply]
@KonstantinKlepikov, yes, that is what I want!

**Reply by interob** [reply]
I would say hiding/unhiding could be the same as collapse/restore, depending on the looks of its implementation

**Reply by MarkBennett** [reply]
I see that Shortcut now has a setting to automatically collapse empty columns. Would be awesome to see GitHub projects pick this up as well.

**Comment by aspearman** [comment, marked as answer]
[The latest GitHub Issues update](https://github.blog/changelog/2022-09-02-the-new-github-issues-september-2nd-update/#🙈-auto-hide-columns-with-board-filters) includes a change which **hides a column if there are no issues** matching the filter options.

![](https://i0.wp.com/user-images.githubusercontent.com/102542428/188190593-ef75e9eb-e1a1-42a2-bf5d-6ec29d69ad72.gif?ssl=1)

That's great and solves my particular annoyance. While I could see why people would want a little chevron somewhere to minimize columns even with cards in them, that's actually not what this discussion was about so I'll mark this as answered/solved.

**Reply by MarkBennett** [reply]
Thank you! This is great news! 😄

**Reply by interob** [reply]
This is very nice! Would be nicer even if users have an option to toggle visibility on their own judgement as well -- a column being empty as a condition for this functionality to work also is a missed opportunity in a way. Kindly up vote #15588 if you feel you could use a little show/hide toggle button too.
